openstack/neutron/templates/daemonset-network_agents.yaml

```diff
@@ -0,0 +1,194 @@
+{{ if $.Values.agent.enable_daemonset }}
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: neutron-agents
+  labels:
+    system: openstack
+    type: backend
+    component: neutron
+spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        name: neutron-agents
+        system: openstack
+        type: backend
+        component: neutron
+    spec:
+      hostNetwork: true
+      hostPID: true
+      hostIPC: true
+      nodeSelector:
+        species: network
+      tolerations:
+        - key: "species"
+          operator: "Equal"
+          value: "network"
+          effect: "NoSchedule"
+      containers:
+        - name: neutron-dhcp-agent
+          image: {{.Values.global.imageRegistry}}/{{.Values.agent.image_name}}:{{.Values.agent.image_tag}}
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          command:
+            - bash
```

Comment:
Personally, I am more a fan of executing the script directly,
and setting the permissions on the container to executable.
But that is just a matter of taste.